app/config.go

```diff
@@ -485,50 +519,59 @@ func NewHostedChains() *types.HostedBlockchains {
 		// if does not exist create one
 		jsonFile, err = os.OpenFile(chainsPath, os.O_RDWR|os.O_CREATE, os.ModePerm)
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// generate hosted chains from user input
 		c := GenerateHostedChains()
 		// create dummy input for the file
 		res, err := json.MarshalIndent(c, "", "  ")
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// write to the file
 		_, err = jsonFile.Write(res)
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
 		}
 		// close the file
 		err = jsonFile.Close()
 		if err != nil {
-			panic(NewInvalidChainsError(err))
+			fmt.Println(NewInvalidChainsError(err))
+			os.Exit(1)
```

Comment:
```suggestion
		        log.Fatal(NewInvalidChainsError(err))
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work